src/modules/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,137 @@
+import React, {
+  useRef,
+  PropsWithChildren,
+  useImperativeHandle,
+  RefObject,
+  useMemo
+} from "react";
+import Popover, {
+  IPopoverRef,
+  IProps as IPopoverProps
+} from "../Popover/Popover";
+import { ThemeStyleSheetFactory, WithStylesProps } from "../..";
+import { withStyles } from "../../base";
+
+export interface IItem {
+  id: string;
+  text: string;
+  inactive?: boolean;
+}
+
+export interface IProps<T extends IItem>
+  extends PropsWithChildren<IPopoverProps> {
```

Comment:
Have you already considered the case of a disabled dropdown?

Reply:
Added to popover